### PR TITLE
Adds slider component for preferences, converts TTS volume, and pitch to use it

### PR DESCRIPTION
**Pull request by peptron1**
## About The Pull Request

I want to add volume sliders, so I made a slider component.

## Why It's Good For The Game

In my opinion, volume sliders should probably be, uh, sliders.

## Changelog
:cl:
qol: TTS volume preference is not actually a volume slider, instead of a volume number input.
/:cl:

### CODE REVIEW

**Comment by TheVekter** [comment]
Peptron comes back 3 years later and drops what might be the best PR of the month. Wild.

**Comment by peptron1** [comment]
Now there are 2 sliders!
(linter please pass)

File: .vscode/settings.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
 	},
 	"[typescriptreact]": {
 		"editor.rulers": [80],
-		"editor.defaultFormatter": "esbenp.prettier-vscode",
+		"editor.defaultFormatter": "vscode.typescript-language-features",
```
Comment (Mothblocks) [inline .vscode/settings.json:35]: did someone tell you to change this

Comment (peptron1) [reply .vscode/settings.json:35]: no, It was just changed after I ran the prettier thing I think?

**Pull request merged**